packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -543,7 +543,12 @@ export class MgtPeoplePicker extends MgtTemplatedTaskComponent {
   };
 
   private get hasMaxSelections(): boolean {
-    return this.selectionMode === 'single' && this.selectedPeople.length >= 1;
+    return (
+      this.selectionMode === 'single' &&
+      (this.selectedPeople.length >= 1 ||
```

Comment:
This change creates an assumption that the id provided resolves to a user or group. 

I'd play a little safer and leave this code here as is and move the check in loadState